ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/client/Web3JExecutionEngineClient.java

```diff
@@ -62,6 +66,9 @@ public Web3JExecutionEngineClient(
       final String eeEndpoint,
       final TimeProvider timeProvider,
       final Optional<JwtConfig> jwtConfig) {
+    SimpleModule module = new SimpleModule("TekuEESsz", new Version(1, 0, 0, null, null, null));
+    module.addSerializer(SignedBeaconBlock.class, new SignedBeaconBlockSerializer());
```

Comment:
Probably not needed here so not worth it, but if we wind up needing to add multiple classes that use TypeDefinition to serialise I wonder if we can do a generic `module.addSerializer(SszData.class` and it would pick up all SSZ objects.

Reply:
it was my first approach but seems to be tricky:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/15999009/157392210-038939de-ed06-4291-a7d6-7bb71bb52c33.png">

Reply:
oh yeah, the captures cause  a bunch of fun. I think I'd just cast as required and suppress warnings.  We know that if you get the schema from an object you can use that schema to deserialise the object itself.

Actually I wonder if we could just add an `SszData.jsonSerialize()` method that could do it internally and then can probably use generics to make it all work.  Not needed for this PR though.